app/src/main/java/com/github/siela1915/bootcamp/ZoomOutPageTransformer.java

```diff
@@ -0,0 +1,44 @@
+package com.github.siela1915.bootcamp;
+
+import android.view.View;
+
+import androidx.viewpager2.widget.ViewPager2;
+
+public class ZoomOutPageTransformer implements ViewPager2.PageTransformer {
+    private static final float MIN_SCALE = 0.85f;
+    private static final float MIN_ALPHA = 0.5f;
+
+    public void transformPage(View view, float position) {
+        int pageWidth = view.getWidth();
+        int pageHeight = view.getHeight();
+
+        if (position < -1) { // [-Infinity,-1)
+            // This page is way off-screen to the left.
+            view.setAlpha(0f);
+
+        } else if (position <= 1) { // [-1,1]
```

Comment:
else if statement only checks if position == 1 which will rarely happen since position is a float.
I'm not sure this is the interval you meant to test.

Reply:
It tests the interval [-1, 1], meaning that the view is at least a bit visible on the screen